spec/features/admin_logins_spec.rb

```diff
@@ -81,7 +82,7 @@
       fill_in 'user[password]', with: user.password
       click_link_or_button('Log in')
 
-      expect(current_path).to eq(root_path)
+      expect(current_path).to eq(company_path(user.company.id))
```

Comment:
The association between users and companies isn't built into the factories so I created the user separately.  If this becomes a burden later, I'll add the association to the factory.